app/assets/javascripts/redux/modules/models/responses.js

```diff
@@ -64,4 +65,15 @@ export const reducer = handleActions({
 
   [LISTEN_SIGNALS_POST_REQUEST_SUCCESS]: handleResponseRequest,
 
+  [LISTEN_SIGNALS_DELETE_REQUEST_SUCCESS]: (state, action) => {
+    const responses = action.meta.signal.responses;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/15848772/19137512/5bcfe192-8b42-11e6-84e2-9052aab06751.png)

`responses` is already an array of ids. You shouldn't need to do any mapping here.